code/game/objects/structures/vehicles/wheelchair.dm

```diff
@@ -207,7 +207,9 @@
 /obj/structure/bed/chair/vehicle/wheelchair/motorized/syndicate
 	nick = "medical malpractice"
 	desc = "A chair with fitted wheels which is powered by an internal cell. It seems to ride higher than other wheelchairs."
-	default_cell_path = /obj/item/weapon/cell/hyper
+
+/obj/structure/bed/chair/vehicle/wheelchair/motorized/syndicate/getMovementDelay()
+	return (..() + 1) //Somewhat slower
```

Comment:
My saycode rewrite, its one of your most expensive procs and I optimized it 10 times over